instrumentation/instabeego/version.go

```diff
@@ -0,0 +1,10 @@
+// (c) Copyright IBM Corp. 2023
+// (c) Copyright Instana Inc. 2023
+
+//go:build go1.18
+// +build go1.18
+
+package instabeego
+
+// Version is the instrumentation module semantic version
+const Version = "0.1.0"
```

Comment:
You can leave the version as 0.0.0, as the GH Action will take care of increasing the version appropriately.
Just make sure to choose "minor version" when triggering the release Action :) We can go through it together when the time comes.